config/locales/views/submissions/en.yml

```diff
@@ -63,6 +63,7 @@ en:
       remark_requested: Remark Requested
     status: Status
     student:
+      create_link: Create Link
```

Comment:
I don't love "Create Link".  I don't think that it really conveys the intended use of "submit a file or video hosted at a different url". But let's discuss this at a meeting to figure out the best way to go with this

Reply:
From the meeting:

- change "Create Link" to "Submit Link"
- change "Upload File" to "Submit File"